docs/ManualDeploymentGuideScalarDBServerOnAKS.md

```diff
@@ -1,178 +1,72 @@
-# Deploy Scalar DB Server on Azure
+# Deploy Scalar DB Server on AKS
```

Comment:
@feeblefakie  
The official name of EKS is `Amazon Elastic Kubernetes Service`.
So, I think there is no problem if we state it `Amazon EKS`.

However, the official name of AKS is `Azure Kubernetes Service`.
So, I think it is not good if we state it `Azure AKS`. (It states like `Azure Azure Kubernetes Service`...)

Therefor, I think it is better to describe official name as follows in the title.
```
# Deploy Scalar DB Server on Amazon EKS (Amazon Elastic Kubernetes Service)
# Deploy Scalar DB Server on AKS (Azure Kubernetes Service)
```

Also, I think there is no problem to use abbreviated names (i.e., `EKS` and `AKS`) in the body (other that tilte) of document.

What do you think?

---

For your reference, in the previous documents, they use `EKS and DynamoDB` or `ASK and Cosmos DB`.
So, I think there is no problem even if the titles are as follows (using Cloud Service Name).
```
# Deploy Scalar DB Server on AWS
# Deploy Scalar DB Server on Azure
```

However, in the new document, I will describe that **"You can use any databases that supported by Scalar DB"** (I think using kubernetes service and database in the different cloud service is very rare case, but this document does not force using database on the same cloud service as kubernetes service).
So, I think it is better to use **Kubernetes Service Name** as a title instead of Cloud Service Name.

Reply:
It sounds good to me. Thank you!